Modules/ITS/itsTrack.json

```diff
@@ -45,9 +45,24 @@
           "nBCbins": "103",
           "dicttimestamp" : "0",
           "doNorm" : "1",
-          "InvMasses" : "0"
+          "InvMasses" : "0",
+          "UseDefaultMomResPar" : "0",
```

Comment:
Can you please add an option to enable/disable your new plots? Default is 1 (i.e. enabled).